R/utils.R

```diff
@@ -372,16 +372,74 @@ markdown_to_text <- function(text) {
     unname()
 }
 
-# Get prepending and appending text based on a simple pattern
-get_pre_post_txt <- function(pattern) {
+#' Handle formatting of a pattern in a \code{fmt_*()} function
+#'
+#' Within the context of a \code{fmt_*()} function, we always have the
+#' single-length character vector of \code{pattern} available to describe a
+#' final decoration of the formatted values. We use \pkg{glue}'s semantics here
+#' (and literally, it's \code{glue::glue()}) and reserve \code{x} to be the
+#' formatted values. We can use \code{x} multiple times in the pattern or even
+#' variables in global scope or R statements within curly braces.
+#' @param pattern An expression string that is the primary input to
+#'   \code{glue::glue()}). The variable \code{x} is reserved and represents the
+#'   formatted values.
+#' @param values The values (as a character vector) that are formatting within
+#'   the \code{fmt_*()} function.
+#' @importFrom glue glue
+#' @noRd
+apply_pattern_fmt_x <- function(pattern,
+                                values) {
 
-  prefix <- strsplit(pattern, "\\{x\\}")[[1]][1]
-  suffix <- strsplit(pattern, "\\{x\\}")[[1]][2]
+  glue::glue(pattern, x = values) %>% as.character()
+}
 
-  prefix <- ifelse(is.na(prefix), "", prefix)
-  suffix <- ifelse(is.na(suffix), "", suffix)
+# This function normalizes the `suffixing` input to a
+# character vector which is later appended to scaled
+# numerical values; the input can either be a single
+# logical value or a character vector
+normalize_suffixing_inputs <- function(suffixing) {
```

Comment:
What's happening here, did this just move from a different spot?

Reply:
It's now moved to line 400 (compared to line 546 in master). I carefully checked to see if this is different from master and found it's identical.